nifi-registry/nifi-registry-core/nifi-registry-utils/src/main/java/org/apache/nifi/registry/util/FileUtils.java

```diff
@@ -38,7 +38,9 @@
  * A utility class containing a few useful static methods to do typical IO
  * operations.
  *
+ * @deprecated Use {@link org.apache.nifi.util.file.FileUtils}
  */
+@Deprecated
 public class FileUtils {
```

Comment:
If there is no other place where it is being used, I think it might be the subject of remoeval (Not part of the API, is not exposed and we expect the same behaviour without it)